testpmd/hooks/templates/sriov-network.yml.j2

```diff
@@ -7,6 +7,6 @@ spec:
   networkNamespace: "{{ cnf_namespace }}"
   vlan: {{ sriov['vlan']|int }}
   resourceName: "{{ sriov['resource'] }}"
-  spoofChk: "on"
-  trust: "on"
-  ipam: '{ }'
+  spoofChk: "{{ sriov['spoofchk'] | default('off') }}"
```

Comment:
From I understood of the last sync, spoofChk=off and trust=on is for DPDK workload but we want the 'opposite' for the blueprint spoofChk=on and trust=off, isn't it?

Reply:
I'll ask Franck, my understanding was that we wanted `spoofChk=off` with our example-cnf , but I'll check, it wasn't clear to me if we were going to test both DPDK and Kernel workloads,

Reply:
I clarified this point, we want two workloads:
- one with DPDK (example-cnf) using `spoofChk=off and trust=on`
- another workload for Kernel using `spoofChk=on and trust=off` (we'll deploy some statefulapp hook or something else)